e2e/nomostest/webhook.go

```diff
@@ -44,39 +43,29 @@ func StopWebhook(nt *NT) {
 		return
 	}
 	webhookName := configuration.Name
-	webhookGK := "validatingwebhookconfigurations.admissionregistration.k8s.io"
+	webhookGK := "ValidatingWebhookConfiguration"
```

Comment:
You can use `kinds.ValidatingWebhookConfiguration().Kind` instead.

Reply:
Done